### PR TITLE
add verbose to options, console.log on if verbose

### PR DESCRIPTION
**Pull request by randyhoulahan**

### CODE REVIEW

File: src/directives/vue.js [hunk 1]
```diff
@@ -69,7 +71,7 @@ function register(ngModule) {
 
         _.forEach(syncedPropertiesMapping, (ngProp, vueProp) => { // .sync
           vm.$children.forEach((c) => c.$on(`update:${vueProp}`, (value) => {
-            console.log(`vue(${vueProp}) => ng(${ngProp}) =`, value);
+            if(options.verbose) log(`vue(${vueProp}) => ng(${ngProp}) =`, value);
```
Comment (sbilodeau) [inline src/directives/vue.js:74]: You dropped `console` by mistake in from of `log`

**Review by sbilodeau — CHANGES_REQUESTED** [review]
There is an error. See specific comment on line

File: src/directives/vue.js (hunk 1 above)
Comment (sbilodeau) [reply src/directives/vue.js:74]: BTW thank you for doing that i wanted to do it long time ago

Comment (randyhoulahan) [reply src/directives/vue.js]: done

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**